### PR TITLE
make emps affect the other zlevels

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Emps will now affect other z levels too

## Why It's Good For The Game
More features for multiz maps, emps are no longer safe when used on other station levels

## Changelog

:cl:
balance: emps will now affect other z levels too
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
Emps are already really fucking annoying and difficult to stop grief with, this does not sound fun.

File: code/game/objects/empulse.dm [hunk 1]
```diff
@@ -15,18 +16,42 @@
...
+		light_range *= 0.25
+		var/turf/above_level = SSmapping.get_turf_above(new_location_above)
+		var/turf/below_level = SSmapping.get_turf_below(new_location_below)
+
+		if(above_level && isturf(above_level))
+			empulse_call(above_level, heavy_range, light_range)
+			new_location_above = above_level
+		if(below_level && isturf(below_level))
+			empulse_call(below_level, heavy_range, light_range)
+			new_location_below = below_level
```
Comment (Watermelon914) [inline code/game/objects/empulse.dm:37]: maybe considering decreasing the intensity as it travels through zlevels by some coefficient

File: code/game/objects/empulse.dm [hunk 2]
```diff
@@ -15,18 +16,42 @@
...
+
+	return 1
```
Comment (Watermelon914) [inline code/game/objects/empulse.dm:39]:
```suggestion
	return TRUE
```

File: code/game/objects/empulse.dm (hunk 1 above)
Comment (Ghilker) [reply code/game/objects/empulse.dm:37]: `heavy_range *= 0.25`
`light_range *= 0.25`
already done that

**Comment by optimumtact** [comment]
again, I don't think cross z level interactions are good except in the case network connections.

**Pull request closed without merging**